ApplicationCode/ProjectDataModel/CMakeLists_files.cmake

```diff
@@ -169,6 +169,13 @@ ${CMAKE_CURRENT_LIST_DIR}/RimFractureModelStressCurve.h
 ${CMAKE_CURRENT_LIST_DIR}/RimAbstractPlotCollection.h
 ${CMAKE_CURRENT_LIST_DIR}/RimFractureModelPropertyCurve.h
 ${CMAKE_CURRENT_LIST_DIR}/RimFaciesProperties.h
+${CMAKE_CURRENT_LIST_DIR}/RimNonNetLayers.h
```

Comment:
I think the fracture model files must be moved into a separate folder. This can be done in a separate PR after this one.

Reply:
Added #6820 to remember to do this.